src/components/Tab/src/styles/tab.css

```diff
@@ -0,0 +1,25 @@
+.denhaag-tabs__tab {
+  padding-block-start: var(--denhaag-tabs-tab-padding-block-start);
+  padding-inline-start: var(--denhaag-tabs-tab-padding-inline-start);
+  padding-block-end: var(--denhaag-tabs-tab-padding-block-end);
+  padding-inline-end: var(--denhaag-tabs-tab-padding-inline-end);
+  color: var(--denhaag-tabs-tab-color);
+  font-size: var(--denhaag-tabs-tab-font-size);
+  font-weight: var(--denhaag-tabs-tab-font-weight);
+  font-family: var(--denhaag-tabs-tab-font-family);
+  line-height: 1.5;
+}
+
+.denhaag-tabs__tab.denhaag-tabs__tab--selected {
+  font-weight: var(--denhaag-tabs-tab-selected-font-weight);
+}
+
+.denhaag-tabs__tab:focus:after {
```

Comment:
I would like to see `.denhaag-tabs__tab--focus:after` added here for automated testing purposes.

Reply:
Added! Thanks